requirements/_requirements.txt

```diff
@@ -21,4 +21,5 @@ pytest-asyncio<=0.21.1
 networkx>=3.1
 pydantic>=2.0
 openai>=1.12.0
-structlog>=24.1.0
\ No newline at end of file
+structlog>=24.1.0
+zxing-cpp>=2.2.0
```

Comment:
@PawelPeczek-Roboflow - is this the correct requirements.txt file for this dependency?

Reply:
I would say so, at least for now we do not have a separate file with requirements. 
Also checked that lib is quite popular and has no security issues. Shall be fine to add.